src/OrleansProviders/ConfigExtensions.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System.Reflection;
+
+
+namespace Orleans.Runtime.Configuration
+{
+    /// <summary>
+    /// Extension methods for configuration classes specific to OrleansProviders.dll 
+    /// </summary>
+    public static class CongigExtensions
+    {
+        /// <summary>
+        /// Adds default storage provider named "MemoryStore" of type Orleans.Storage.MemoryStorage
+        /// </summary>
+        /// <param name="config">Cluster configuration object to add provider to</param>
+        /// <returns></returns>
+        public static ClusterConfiguration AddDefaultMemoryStore(this ClusterConfiguration config)
+        {
+            config.Globals.RegisterStorageProvider<Storage.MemoryStorage>("MemoryStore");
+            return config;
```

Comment:
For example, people could argue that if it's fluent, it should be `.WithMemoryStorage()`, or 10 different other options